job_vars/cinder-iscsi.yaml

```diff
@@ -292,11 +292,11 @@ services:
     template: cinder-backup.conf
 
 
-win_cherry_picks:
-  - project: openstack/neutron
-    path: "{{ win_dir.build }}\\neutron"
-    patches:
-      - refs/changes/59/763159/1
+# win_cherry_picks:
+  # - project: openstack/neutron
```

Comment:
Nit: we might as well remove this entry, keeping just the compute-hyperv one as a sample.

Reply:
cleaned all outdated cherry picks, still have sample in the os-brick-fc.yaml